cmd/podman/volumes/rm.go

```diff
@@ -73,6 +73,9 @@ func rm(cmd *cobra.Command, args []string) error {
 			fmt.Println(r.Id)
 		} else {
 			setExitCode(r.Err)
+			if rmOptions.Force && registry.GetExitCode() == 1 {
```

Comment:
This should only happen if the error is volume does not exist, and this is the only error returned.